### PR TITLE
Make repository compliant with REUSE license checker

### PR DESCRIPTION
**Pull request by oddhack**
Add SPDX identifiers and REUSE (https://reuse.software/) infrastructure.
To check compliance when adding new files, install REUSE and run "reuse
lint" in the repository root. This could be easily added to the travis
CI configuration, as well.

For files with no copyright listed, I assigned copyright to Khronos rather than @MarkCallow personally. If that's not correct then please push an update to the branch before merging. For licensing purposes only the license matters, not the copyright holder.

### CODE REVIEW

File: .reuse/dep5 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: dfdutils
+Source: https://github.com/KhronosGroup/dfdutils
+
+Files: KHR/khr_df.h
+Copyright: 2015-2019 The Khronos Group Inc.
+License: MIT
```
Comment (MarkCallow) [inline .reuse/dep5:7]: This is a Khronos header in the Data Formats repo. We should change its license to Apache 2.0.

**Comment by MarkCallow** [comment]
Thank you for a very thorough PR. I've been discussing with Andrew & Neil and I think we're going to ask you to change the license in khr_df.h to Apache 2.0 over in its DataFormat repo source. I'd like to get that resolved then update this PR before merging.

**Comment by MarkCallow** [comment]
I've updated this repo to the re-licensed khr_df.h. @oddhack please update this PR.

**Comment by oddhack** [comment]
I believe this is ready to merge now. Once that's done, please update the KTX-Software repo's copy of dfdutils and khr_df.h as well, then we can get REUSE working there as well.

**Comment by MarkCallow** [comment]
Why does LICENSE.adoc give links to the original license test rather than the copies in LICENSES? Another way of putting the question is why bother with the LICENSES directory if all used licenses are available on line?

**Comment by oddhack** [comment]
LICENSES is required by REUSE while github practice encourages LICENSE. Perhaps eventually there will be more convergence; it is slightly unfortunate to have the duplication, although many licenses require a complete copy be included with "the Work" anyway.

**Comment by MarkCallow** [comment]
Could we change the links in LICENSE.adoc to point to the copies in LICENSES.txt? It's probably not worth it though.

I have just created a PR to pull these changes into KTX-Software. I'm waiting for the CI builds.

**Pull request merged**